### PR TITLE
Apple.Safari version 4.42

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/99469)

### CODE REVIEW

File: manifests/a/Apple/Safari/4.42/Apple.Safari.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: Apple.Safari
+PackageVersion: 4.42
+Installers:
+- InstallerUrl: http://appldnld.apple.com.edgesuite.net/content.info.apple.com/Safari3/061-4833.20080619.E45tg/SafariSetup.exe
+  Architecture: x86
+  InstallerType: exe
+  InstallerSha256: FE14C852F90CE7161475CB5EBCBC70F53B2C0F22AC594566820550BD281831D5
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
```
Comment (Trenly) [inline manifests/a/Apple/Safari/4.42/Apple.Safari.installer.yaml:13]:
```suggestion
    Silent: /qn
    SilentWithProgress: /qb
```

image

File: manifests/a/Apple/Safari/4.42/Apple.Safari.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Apple.Safari
+PackageVersion: 4.42
+Installers:
+- InstallerUrl: http://appldnld.apple.com.edgesuite.net/content.info.apple.com/Safari3/061-4833.20080619.E45tg/SafariSetup.exe
```
Comment (Trenly) [inline manifests/a/Apple/Safari/4.42/Apple.Safari.installer.yaml:7]:
```suggestion
- InstallerUrl: http://appldnld.apple.com/Safari3/061-4833.20080619.E45tg/SafariSetup.exe
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Does this package have a dependency?
image

**Comment by stephengillie** [comment]
> Does this package have a dependency? image

I see this in a Windows 10 sandbox but not a Windows 11 VM, thus manual validation could easily go either way. Should OS market share inform the decision?

**Comment by upintheairsheep** [comment]
> > Does this package have a dependency? image
> 
> I see this in a Windows 10 sandbox but not a Windows 11 VM, thus manual validation could easily go either way. Should OS market share inform the decision?

No, just make this Windows 11 or higher and add dependencies later when it releases.

**Comment by upintheairsheep** [comment]
@stephengillie again, until dependency support is added, let’s make this Windows 11 or higher.

File: manifests/a/Apple/Safari/4.42/Apple.Safari.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Apple.Safari
+PackageVersion: 4.42
+Installers:
```
Comment (stephengillie) [inline manifests/a/Apple/Safari/4.42/Apple.Safari.installer.yaml:6]:
```suggestion
MinimumOSVersion: 10.0.22000.0
Installers:
```

File: manifests/a/Apple/Safari/4.42/Apple.Safari.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+PackageName: Safari
+PackageUrl: http://apple.com/Safari
+License: Proprietary
+LicenseUrl: https://www.apple.com/legal/internet-services/terms/site.html
+CopyrightUrl: http://apple.com/legal
+ShortDescription: Safari is the best way to experience the internet on all your Apple devices. It brings robust customization options, powerful privacy protections, and optimizes battery life — so you can browse how you like, when you like.
+Description: Safari is the best way to experience the internet on all your Apple devices. It brings robust customization options, powerful privacy protections, and optimizes battery life — so you can browse how you like, when you like. And when it comes to speed, it’s the world’s fastest browser. With a blazing-fast JavaScript engine, Safari is the world’s fastest browser.1 It’s developed to run specifically on Apple devices, so it’s geared to make the most out of your battery life and deliver long-lasting power. And with Apple silicon, it’s even faster than ever before.2
+Tags:
+- browser,apple
+ReleaseNotes: n
```
Comment (ItzLevvie) [inline manifests/a/Apple/Safari/4.42/Apple.Safari.locale.en-US.yaml:20]:
```suggestion
```

File: manifests/a/Apple/Safari/4.42/Apple.Safari.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+PrivacyUrl: http://apple.com/privacy
+PackageName: Safari
+PackageUrl: http://apple.com/Safari
+License: Proprietary
+LicenseUrl: https://www.apple.com/legal/internet-services/terms/site.html
+CopyrightUrl: http://apple.com/legal
+ShortDescription: Safari is the best way to experience the internet on all your Apple devices. It brings robust customization options, powerful privacy protections, and optimizes battery life — so you can browse how you like, when you like.
+Description: Safari is the best way to experience the internet on all your Apple devices. It brings robust customization options, powerful privacy protections, and optimizes battery life — so you can browse how you like, when you like. And when it comes to speed, it’s the world’s fastest browser. With a blazing-fast JavaScript engine, Safari is the world’s fastest browser.1 It’s developed to run specifically on Apple devices, so it’s geared to make the most out of your battery life and deliver long-lasting power. And with Apple silicon, it’s even faster than ever before.2
+Tags:
+- browser,apple
```
Comment (ItzLevvie) [inline manifests/a/Apple/Safari/4.42/Apple.Safari.locale.en-US.yaml:19]:
```suggestion
- browser
- apple
```

File: manifests/a/Apple/Safari/4.42/Apple.Safari.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+PublisherUrl: http://apple.com
+PublisherSupportUrl: http://support.apple.com/
+PrivacyUrl: http://apple.com/privacy
+PackageName: Safari
+PackageUrl: http://apple.com/Safari
+License: Proprietary
+LicenseUrl: https://www.apple.com/legal/internet-services/terms/site.html
+CopyrightUrl: http://apple.com/legal
+ShortDescription: Safari is the best way to experience the internet on all your Apple devices. It brings robust customization options, powerful privacy protections, and optimizes battery life — so you can browse how you like, when you like.
+Description: Safari is the best way to experience the internet on all your Apple devices. It brings robust customization options, powerful privacy protections, and optimizes battery life — so you can browse how you like, when you like. And when it comes to speed, it’s the world’s fastest browser. With a blazing-fast JavaScript engine, Safari is the world’s fastest browser.1 It’s developed to run specifically on Apple devices, so it’s geared to make the most out of your battery life and deliver long-lasting power. And with Apple silicon, it’s even faster than ever before.2
```
Comment (ItzLevvie) [inline manifests/a/Apple/Safari/4.42/Apple.Safari.locale.en-US.yaml:17]:
```suggestion
Description: Safari is the best way to experience the internet on all your Apple devices. It brings robust customization options, powerful privacy protections, and optimizes battery life — so you can browse how you like, when you like. And when it comes to speed, it’s the world’s fastest browser. With a blazing-fast JavaScript engine, Safari is the world’s fastest browser.1 It’s developed to run specifically on Apple devices, so it’s geared to make the most out of your battery life and deliver long-lasting power. And with Apple silicon, it’s even faster than ever before.
```

File: manifests/a/Apple/Safari/4.42/Apple.Safari.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Apple.Safari
+PackageVersion: 4.42
```
Comment (ItzLevvie) [inline manifests/a/Apple/Safari/4.42/Apple.Safari.installer.yaml:5]:
```suggestion
PackageVersion: 3.525.21.0
```
This must match Control Panel.

File: manifests/a/Apple/Safari/4.42/Apple.Safari.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Apple.Safari
+PackageVersion: 4.42
```
Comment (ItzLevvie) [inline manifests/a/Apple/Safari/4.42/Apple.Safari.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.525.21.0
```
This must match Control Panel.

File: manifests/a/Apple/Safari/4.42/Apple.Safari.yaml [hunk 9]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Apple.Safari
+PackageVersion: 4.42
```
Comment (ItzLevvie) [inline manifests/a/Apple/Safari/4.42/Apple.Safari.yaml:5]:
```suggestion
PackageVersion: 3.525.21.0
```
This must match Control Panel.

File: manifests/a/Apple/Safari/4.42/Apple.Safari.locale.en-US.yaml [hunk 10]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Apple.Safari
+PackageVersion: 4.42
+PackageLocale: en-US
+Publisher: Apple
```
Comment (ItzLevvie) [inline manifests/a/Apple/Safari/4.42/Apple.Safari.locale.en-US.yaml:7]:
```suggestion
Publisher: Apple Inc.
```
This must match Control Panel.

File: manifests/a/Apple/Safari/4.42/Apple.Safari.installer.yaml [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Apple.Safari
+PackageVersion: 4.42
+MinimumOSVersion: 10.0.22000.0
+Installers:
+- InstallerUrl: http://appldnld.apple.com/Safari3/061-4833.20080619.E45tg/SafariSetup.exe
+  Architecture: x86
+  InstallerType: exe
+  InstallerSha256: FE14C852F90CE7161475CB5EBCBC70F53B2C0F22AC594566820550BD281831D5
```
Comment (ItzLevvie) [inline manifests/a/Apple/Safari/4.42/Apple.Safari.installer.yaml:11]:
```suggestion
  InstallerSha256: FE14C852F90CE7161475CB5EBCBC70F53B2C0F22AC594566820550BD281831D5
  ProductCode: "{C9D96682-5A4D-45FA-BA3E-DDCB2B0CB868}"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll also need to change `manifests/a/Apple/Safari/4.42` to `manifests/a/Apple/Safari/3.525.21.0`

**Review by stephengillie — APPROVED** [review]

**Pull request merged**